ci-operator/config/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-main.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: benchmark-runner
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: stress-ng-test
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    workflow: redhat-performance-benchmark-runner-stress-ng-test
```

Comment:
need to add this lower section to this file: https://github.com/openshift/release/blob/7fb34641b9830120898d63913c0ed0930d2ea760/ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main.yaml#L18C1-L21C28